core/arch/arm/plat-stm/tz_a9init.S

```diff
@@ -368,10 +368,10 @@ loop_1:
 	movt r1, #(CPU_PORT_FILT_START >> 16)
 	str  r1, [r0, #PL310_ADDR_FILT_START]
 
-	/* Allow NSec to manage FIQ/Imprecise abort */
-	mrc p15, 0, r0, c1, c1, 0    /* read Secure Configuration Register */
-	orr r0, r0, #0x30            /* SCR[FW]=1, SCR[AW]=1 */
-	mcr p15, 0, r0, c1, c1, 0    /* write updated value in Secure Configuration Register */
+	/* Allow NSec to manage FIQ/Imprecise abort (SCR[FW]=1, SCR[AW]=1) */
+	mrc p15, 0, r0, c1, c0, 0
+	orr r0, r0, #0x30            /*  */
+	mcr p15, 0, r0, c1, c0, 0    /* write updated value in Secure Configuration Register */
```

Comment:
What?! It's either correct or incorrect, isn't it?
Now, it seems we're close to having a working plat-stm again, so I would like a commit (one commit) that fixes the issue, and I would like it to be tested and reviewed today so that I can merge it in release 2.2.0. Can you do this?